### PR TITLE
Updated xUnit test reporter

### PR DESCRIPTION
**Pull request by sixsixtrample**
Added check if test.err exists and changed test for whether a test has
passed or failed. Tested both a test that should pass and a test that
should fail. Both printed correctly.

### CODE REVIEW

File: lib/reporter/xunit.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ function XUnit(options) {
         parentSuite.pending++;
         test.id = ++id;
         test.time = new Date() - tests[test.pid][0].end,
```
Comment (christian-bromann) [inline lib/reporter/xunit.js]: could you replace the `,` with an `;` and remove the indentation below?

**Comment by christian-bromann** [comment]
@sixsixtrample  One comment on something that is not related to this but would be great if we can fix this as well. After that it is good to go

File: lib/reporter/xunit.js (hunk 1 above)
Comment (sixsixtrample) [inline lib/reporter/xunit.js]: Updated.

Comment (christian-bromann) [inline lib/reporter/xunit.js]: @sixsixtrample I still see
```
test.time = new Date() - tests[test.pid][0].end,
```
but it should be
```
test.time = new Date() - tests[test.pid][0].end;
```

File: lib/reporter/xunit.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ function XUnit(options) {
         parentSuite.passes++;
         test.id = ++id;
         test.time = new Date() - tests[test.pid][0].end,
-        test.timestamp = tests[test.pid][0].end = new Date();
+            test.timestamp = tests[test.pid][0].end = new Date();
         test.runnerData = runnerData[test.pid];
```
Comment (christian-bromann) [inline lib/reporter/xunit.js]: please also change it here

File: lib/reporter/xunit.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ function XUnit(options) {
         parentSuite.failures++;
         test.id = ++id;
         test.time = new Date() - tests[test.pid][0].end,
-        test.timestamp = tests[test.pid][0].end = new Date();
+            test.timestamp = tests[test.pid][0].end = new Date();
```
Comment (christian-bromann) [inline lib/reporter/xunit.js]: and here

**Comment by sixsixtrample** [comment]
Ok, I believe they are all fixed.

**Comment by christian-bromann** [comment]
@sixsixtrample please have a look at the _Files Change_ tab. Have you missed to push to Github?

**Comment by sixsixtrample** [comment]
It says I have no uncommitted changes. And I pushed it to my fork. Is there something else I need to do?

**Comment by christian-bromann** [comment]
Thanks!

**Comment by christian-bromann** [comment]
Published with `v3.2.5`

**Pull request merged**